### DISCUSSION TITLE
[LR1121] set high frequency fail.

### DISCUSSION THREAD

**Discussion by klabshowroom** [jgromes/RadioLib#1355] [Q&A]
I am trying to use high frequency as:
int state = radio.begin(2050.0, 250.0, 10, 7, RADIOLIB_LR11X0_LORA_SYNC_WORD_PRIVATE, 10, 9, 0);

and the response is as:
failed, code -8

is there anything wrong for my setting?

**Comment by K4KDR** [comment]
Hello!

From 

https://jgromes.github.io/RadioLib/group__status__codes.html

... we can see that Error-8 is **Invalid Bandwidth**.

From

https://www.semtech.com/products/wireless-rf/lora-connect/lr1121#documentation

... or from the Radiolib source code, you can see that the 2.4 GHz radio uses LoRa BW values that are different from the values that we are accustomed to using for most LoRa hardware.

![Screenshot from 2024-12-20 00-57-50](https://github.com/user-attachments/assets/f4e7be66-bc4a-41f3-b365-3c72d269c637)


Hope that helps!

**Reply by klabshowroom** [reply]
thansk for your response, and will try it.
